hydra-tui/test/Hydra/TUISpec.hs

```diff
@@ -180,12 +181,17 @@ setupNodeAndTUI' lovelace action =
         chainConfig <- chainConfigFor Alice tmpDir nodeSocket [] tuiContestationPeriod
         -- XXX(SN): API port id is inferred from nodeId, in this case 4001
         let nodeId = 1
+        -- create user key used for committing to a Head
+        let externalKeyFilePath = tmpDir </> "external.skey"
+        externalSKey <- createAndSaveSecretKey externalKeyFilePath
+        let externalVKey = getVerificationKey externalSKey
+
         withHydraNode (contramap FromHydra tracer) chainConfig tmpDir nodeId aliceSk [] [nodeId] hydraScriptsTxId $ \HydraClient{hydraNodeId} -> do
           -- Fuel to pay hydra transactions
           seedFromFaucet_ node aliceCardanoVk lovelace Fuel (contramap FromFaucet tracer)
           -- Some ADA to commit
-          -- TODO: use commit from external wallet in TUI
-          seedFromFaucet_ node aliceCardanoVk 42_000_000 Normal (contramap FromFaucet tracer)
+          seedFromFaucet_ node externalVKey 42_000_000 Normal (contramap FromFaucet tracer)
```

Comment:
Could: move this just after creating the file to indicate it really is not dependent on the `hydra-node` running